src/components/PageControl.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
+import * as React from 'react';
+import { View, StyleSheet, TouchableWithoutFeedback } from 'react-native';
+
+import withTheme from '../core/withTheme';
+import type { Theme } from '../types/Theme';
+
+type Props = {
+  theme: Theme,
+  currentPage: number,
+  numberOfPages: number,
+  hidesForSinglePage?: boolean,
+  pageIndicatorTintColor?: string,
+  currentPageIndicatorTintColor?: string,
+  updateCurrentPageDisplay: number => void,
+  size?: number,
+};
+
+class PageControl extends React.Component<Props> {
+  makeDotStyles = () => {
+    const { theme } = this.props;
+    const size = this.props.size || 8;
```

Comment:
add size to destructuring one line above

Reply:
Done!